bin/check-zookeeper-cluster.rb

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env ruby
+#  encoding: UTF-8
+#
+#  check-zookeeper-cluster
+#
+# DESCRIPTION:
+#   Check if a exhibitor managed Zookeeper cluster is OK.
+#
+#   This check will get exhibitor status information,
+#   and check each seperate node for ruok.
+#   This check also will compare cluster node count with a spesific value,
+#   avg latency from mntr for a threshold value, and If the cluster
+#   has a leader.
+#
+# PLATFORMS:
+#   All
+#
+# DEPENDENCIES:
+#   gem: sensu-plugin
+#
+# USAGE:
+#  Check if a node has Zookeeper running and responds with imok.
+#
+#  Cluster should have 3 nodes, exhibitor status endpoint is not default, and average latency threshold should be 10
+#  ./check-zookeeeper-cluster.rb -c 3 -e http://localhost:8181/exhibitor/v1/cluster/status -l 10
+#
+
+require 'sensu-plugin/check/cli'
+require 'socket'
+require 'net/http'
+require 'json'
+
+class CheckZookeeperCluster < Sensu::Plugin::Check::CLI
+  option :count,
+         description: 'Zookeeper cluster node count',
+         short: '-c count',
+         long: '--count count',
+         default: 2181,
```

Comment:
I am not terribly familiar with zookeeper, is this a typo? This does not seem like a sane default but I might be missing something.